ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -16,12 +17,25 @@ spec:
     - name: verify-ci-results
       image: registry.access.redhat.com/ubi8-minimal
       workingDir: $(workspaces.results.path)
+      env:
+        - name: PREFLIGHT_MIN_VERSION
+          value: $(params.preflight_min_version)
+        - name: PREFLIGHT_RESULTS
+          value: $(params.preflight_results)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+        set -o pipefail
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat $PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
+        if [ "$(printf '%s\n' "$PREFLIGHT_MIN_VERSION" "$PREFLIGHT_VERSION" | sort -V | head -n1)" = "$PREFLIGHT_MIN_VERSION" ]; then
+          echo "Preflight version satisfies the minimun requirement of version"
+        else
+          echo "Preflight is having an outdated version $PREFLIGHT_VERSION"
+          echo "Preflight version requires version $PREFLIGHT_MIN_VERSION or higher."
```

Comment:
Please combine these into a single message: "Preflight version ($PREFLIGHT_VERSION) must be $PREFLIGHT_MIN_VERSION or higher."

Reply:
Updated!